integration-tests/deployment/cs_test.go

```diff
@@ -55,6 +56,10 @@ func TestDeploy(t *testing.T) {
 	deployer := tonChain.Wallet
 	t.Log("Deployer: ", deployer.Address().String())
 
+	// memory environment doesn't block on funding so changesets can execute before the env is fully ready, manually call fund so we block here
```

Comment:
The core version previous was from a custom branch, where the wallet funding exist in environment setup stage. Adding the wallet funding back , as we points to the develop branch, these can be dropped in future PR after custom branch merged